src/commcare_cloud/commands/ansible/service.py

```diff
@@ -428,6 +450,8 @@ def supervisor_process_name(self):
 class Touchforms(SingleSupervisorService):
     name = 'touchforms'
     inventory_groups = ['touchforms']
+    log_location = '/home/cchq/www/{env}/log/formsplayer.log\n' \
+                   '/home/cchq/www/{env}/log/formplayer.clean-{host}.log'
```

Comment:
I believe that touchforms isn't run on any servers anymore, so this could get removed?

Reply:
I can do this: 
https://github.com/dimagi/commcare-cloud/pull/2033/commits/ca88bf55c8a850dc26dfd7dac004b57995676b18
But when I grep for `touchforms` in the repo it still shows up in a bunch of places, maybe a different PR should have a more thorough scrub?

Reply:
Yeah, there's a ticket in the devops backlog to do that: https://trello.com/c/jGzPtKKR/120-remove-references-to-touchforms but I figured it makes sense to remove the reference as we come across it before that is finished

Reply:
good to know about that, I'll be on the lookout